### DISCUSSION TITLE
I have a verified Student account on Github but I get this error: GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

### DISCUSSION THREAD

**Discussion by nishan-chatterjee** [community/community#41378] [GitHub Education] [answered]
When I'm redirected to sign up for Copilot, I checked under:
(VS Code) Subscription Required > (Web) Free Use FAQ > Students and Teachers > Get Student Benefits > Congrats! Your request for [@nishan-chatterjee] was approved.

What should I do?

**Comment by nishan-chatterjee** [comment, marked as answer]
Okay, seems like the problem was with an older VS Code version. I updated VS Code and reinstalled the Copilot extension. Works now.

**Comment by ez2rok** [comment]
I have the same issue as the OP.

I am a student and it says that my student account is recognized: "Congrats! Your request for [@ez2rok] was approved". However, unlike the OP, my VS code is up to date. Here are its specs:
```
Version: 1.74.0 (Universal)
Commit: 5235c6bb189b60b01b1f49062f4ffa42384f8c91
Date: 2022-12-05T16:43:37.594Z (1 wk ago)
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.4.0
Sandboxed: No
```

Nonethless, I got the error message
```
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

Moreover, I got an email from github with the subject line "Your free GitHub Copilot access has expired" telling me I can no longer use Github even though I am still recognized by Github as a student.

Any advice on how to fix this?

**Comment by blakeNaccarato** [comment]
If you're reading this, please see [my comment](https://github.com/orgs/community/discussions/41371#discussioncomment-4403025) over in https://github.com/orgs/community/discussions/41371, a discussion thread with the most votes/activity dealing with this issue. Let's concentrate our discussion there to hopefully gain some traction?

**Reply by nishan-chatterjee** [reply]
@ez2rok, follow @blakeNaccarato's thread. It mentions all the different variants of the issue.
PS: You will probably receive an email saying "Your GitHub Copilot access has been renewed" at some point soon if it's the most common case. In other cases, please follow the thread mentioned.
